apps/src/templates/projects/projectConstants.ts

```diff
@@ -63,3 +65,32 @@ export const convertChannelsToProjectData = function (projects) {
     updatedAt: project.updatedAt,
   }));
 };
+
+export const PROJECT_DEFAULT_IMAGE =
+  '/blockly/media/projects/project_default.png';
+
+export const PROJECT_DEFAULT_CARD_IMAGE_OVERRIDE = {
+  music: musicNoteProjectCardImage,
+};
+
+export const PROJECT_DEFAULT_THUMBNAIL_IMAGE_OVERRIDE: {
+  [projectType: string]: string;
+} = {
+  music: '/shared/images/fill-70x70/courses/logo_music.png',
+};
+
+// Project data from the channels API.
+type project = {
```

Comment:
Hmm, we have a version of this interface in lab2 `types` (where it's called `Channel`) and I'm wondering if we want to reuse/consolidate...ideally it would be great to have a single source of truth for these types (so there's a consistent understanding of what a "project" is across the codebase) but probably out of scope of this PR to do that kind of refactor. Maybe ok to keep here for now but add a comment that this type is duplicated in lab2/types? cc @molly-moen 

also nit: capitalize `Projects`

Reply:
Oh good to know about `Channel` interface - thanks Sanchit! I [updated the interface ](https://github.com/code-dot-org/code-dot-org/pull/58973/commits/71f6bd58a37c1ec1615eb90f334ccc35d943c9b2) adding two optional properties needed for `convertChanneslToProjectData`. @molly-moen lmk if this looks okay to you. If not, I can revert this commit. Thanks!

Reply:
The other option that @sanchitmalhotra126 and I discussed was converting the new `projectUtils.ts` file to a .js file. 🤷‍♀️

Reply:
Seems fine to add those fields there, they already exist on the channel itself, right?

Reply:
Yes, they do. Thanks!